### DISCUSSION TITLE
0.70.4

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9044] [General]
* Work around deadlock in WSL init daemon due to bug in musl libc [GH 8824]
* Fix regression where /etc/wsl.conf interop.enabled setting was not respected [GH 8932]

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/0.70.4'>0.70.4</a>.</em>

**Comment by noknok79** [comment]
Hello,

Is there anyway to install/update to WSL2 0.67 older Microsoft.WSL_0.67.6.0_x64_ARM64.msixbundle using Powershell via Thycotic Administrator, as the only way to install/update. this is the only way to install restricted application.

Thank you!

**Reply by Biswa96** [reply]
Use `Remove-AppxPackage` to remove the current installed WSL Preview version. Then install the version you want with `Add-AppxPackage` command in Powershell. Both requires administrator privilege.
